sdcm/tester.py

```diff
@@ -582,6 +582,8 @@ def is_encrypt_keys_needed(self):
     @teardown_on_exception
     @log_run_info
     def setUp(self):  # pylint: disable=too-many-branches,too-many-statements
+        warnings.filterwarnings(action="ignore", message="unclosed",
```

Comment:
why are you ignoring "unclosed" messages?

Reply:
There was plenty of unclosed resources warnings when running locally on my machine. Warnings were polluting logs making harder to debug. Mostly, warnings were coming from third-party libraries.
This is a duplication of line 113-114 - but placed in setUp() as according to docs, when using unit-test library this warning must be updated for each test (global one does not work). I'll remove original setting as is not needed after this change.

Reply:
should this filter enter the final PR? why? if they are only warnings, we shouldn't filter it out completely

Reply:
somehow we don't see this warnings when executed with hydra anyway.
It's also not a new thing - just moved to more proper place.
These warnings are coming from dependent libraries and it's hard to get anything valuable from them.